atomic_reactor/plugin.py

```diff
@@ -46,6 +47,8 @@ class Plugin(object):
     # by default, if plugin fails (raises exc), execution continues
     is_allowed_to_fail = True
 
+    duration = datetime.timedelta()
```

Comment:
This might be important for an empty plugin - it may not have its duration set and we'll try to work with ```None```.

Not sure if its really needed though, its mostly a safety measure